QRDA/HL7 QRDA I/STU 5.2/v1.1 - Errata/README.md

```diff
@@ -0,0 +1,7 @@
+# HL7 QRDA I STU 5.2 v1.1 errata
+
+Published January 2021
+
+Version 1.1 Added a test enforcing having at most one diagnosis of rank = 1 in Encounter Performed template
+
+When  using the schematron, the vocabulary file voc.xml mudy be placed in the same location as the schematron file.
```

Comment:
s/mudy/must/

Reply:
Corrected the typo.  Committed/pushed the readme.md file to the repo under the current outstanding pr.

Reply:
Note, this typo was present in the v1.0 readme from which this was copied.  That is fixed/committed/pushed as well.